packages/compass-collection-stats/src/components/document-stats-item/document-stats-item.tsx

```diff
@@ -31,8 +28,6 @@ type DocumentStatsItemProps = {
 const DocumentStatsItem: React.FunctionComponent<DocumentStatsItemProps> = ({
   documentCount,
   isTimeSeries,
```

Comment:
This component is essentially empty if `isTimeSeries` is true. For the index stats, we skip them here: https://github.com/mongodb-js/compass/pull/2958/files#diff-f6340fbdf9bd7dca1d628c1ea60ace9041137ca0d43f9e58d2221cc83aaa4212R84

Should the two be skipped the same way, i.e. outside of the component itself?

(We currently still do display the storage size for time series.)